src/redux/comments.js

```diff
@@ -70,7 +70,16 @@ module.exports.commentsReducer = (state, action) => {
             return Object.assign({}, state, {
                 replies: Object.assign({}, state.replies, {
                     // Replies to comments go at the end  of the thread
-                    [action.topLevelCommentId]: state.replies[action.topLevelCommentId].concat(action.comment)
+                    [action.topLevelCommentId]: (
+                        state.replies[action.topLevelCommentId] || []).concat(action.comment)
```

Comment:
I feel like the whole parenthesized expression should be on its own line for readability (assuming our linter allows that)
```suggestion
                    [action.topLevelCommentId]:
                        (state.replies[action.topLevelCommentId] || [])
                            .concat(action.comment)
```